### ISSUE TITLE
whereabouts the issue #50

### ISSUE THREAD

**Issue by masatake** [naoina/toml#57]
Suddenly https://github.com/naoina/toml/issues/50 is gone.
Is it possible to salvage the issue?

**Comment by fjl** [comment]
Interesting! I don't know why the issue has vanished. Are you the submitter of the issue?

**Comment by masatake** [comment]
> Interesting! I don't know why the issue has vanished. Are you the submitter of the issue?

No. I tracked it because the issue included important information about the TOML spec that was not supported by the code in this repository.
